### ISSUE TITLE
Validation: warn if user has form unsupported by Input

### ISSUE THREAD

**Issue by wonder-sk** [lutraconsulting/qgis-mergin-plugin#259]
Some combinations of form setup are not supported by Input and we should try to warn users about that:
- using custom UI file (only auto-generate and drag&drop designer are supported)
- custom python init functions are not supported
- tabs in drag&drop designer are only supported at the top level, and there must not be a mixture of tabs + other widgets

Also some widgets are not supported by Input, which may be also good to note, e.g. classification, color, unique values, json edit, list, key/value, enumeration, binary (blob)

**Comment by Finlay-Belfield** [comment]
@wonder-sk are the customer UI files something that Lutra are working on?

**Comment by wonder-sk** [comment]
@Finlay-Belfield not really. The .ui files are quite tricky to support in the mobile app, as they are aimed for desktop-style user interfaces. If you are considering using .ui files, maybe it would be worth listing features you would like to have in the mobile app, and there may be other options than going through .ui files...

**Comment by Finlay-Belfield** [comment]
reading and writing JSON fields would be very useful
